src/components/Editor/index.jsx

```diff
@@ -265,30 +266,37 @@ const Editor = (
       )}
       <ErrorWrapper {...{ error }} className={errorWrapperClassName}>
         <>
-          <Menu
-            {...{
-              addonCommands,
-              addons,
-              attachmentProps,
-              defaults,
-              editor,
-              editorSecrets,
-              mentions,
-              menuType,
-              neetoKbArticleState,
-              openLinkInNewTab,
-              tooltips,
-              variables,
-            }}
-            className={menuClassName}
-            isIndependant={isMenuIndependent}
-            setIsNeetoKbArticleActive={setNeetoKbArticleState}
-          />
+          {menuType !== "none" && (
+            <Menu
+              {...{
+                addonCommands,
+                addons,
+                attachmentProps,
+                defaults,
+                editor,
+                editorSecrets,
+                mentions,
+                menuType,
+                neetoKbArticleState,
+                openLinkInNewTab,
+                tooltips,
+                variables,
+              }}
+              className={menuClassName}
+              isIndependant={isMenuIndependent}
+              setIsNeetoKbArticleActive={setNeetoKbArticleState}
+            />
+          )}
           {children}
-          <EditorContent
-            className={contentWrapperClassName}
-            {...{ editor, ...otherProps }}
-          />
+          <div style={{ position: "relative" }}>
```

Comment:
@SumitPanda0 This inline style is breaking the entire document editor layout.

<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/66fba15b-983b-43b9-b102-ca167d6b37ee" />

<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/94a3889b-2b47-4c99-b902-1922a6d4dcc2" />